compiler-rt/test/asan/TestCases/Windows/null_deref.cpp

```diff
@@ -1,15 +0,0 @@
-// RUN: %clangxx_asan -O0 %s -o %t && not %run %t 2>&1 | FileCheck %s
-// FIXME: merge this with the common null_deref test when we can run common
-// tests on Windows.
-
-__attribute__((noinline))
-static void NullDeref(int *ptr) {
-  // CHECK: ERROR: AddressSanitizer: access-violation on unknown address
-  // CHECK:   {{0x0*000.. .*pc 0x.*}}
-  ptr[10]++;  // BOOM
-}
-int main() {
-  NullDeref((int*)0);
```

Comment:
What's up with this deleted test?

Reply:
this contained the following comment:
> // FIXME: merge this with the common null_deref test when we can run common
// tests on Windows.

and you'll note that TestCases/null_deref.cpp (the mentioned common test) was also modified to work on windows.

So the deletion is just fixing this fixme :D

Reply:
Right - but I don't see any `XFAIL` being removed from `TestCases/null_deref.cpp`, so to me it seems like that test was already being executed on Windows, with clang - this change just makes it work with MSVC too?

(I.e. in other word, this FIXME that you're removing was already ripe to be removed before this patch.)

Reply:
yep, it was indeed ripe to be removed prior to this patch.

If you like I could extract this change into its own PR I suppose

Reply:
If we started over, this PR could be split into set of PRs where each PR contains same patter of changes. Easier to review.